ramanujantools/cmf/cmf_test.py

```diff
@@ -155,3 +155,26 @@ def test_substitute_trajectory_walk_equivalence():
     assert subbed.walk({n: 1}, iterations, {n: 1}) == unsubbed.walk(
         trajectory, iterations, start
     )
+
+
+def test_delta_correctness():
+    cmf = known_cmfs.hypergeometric_derived_2F1()
+    trajectory = {a: 1, b: 0, c: 1}
+    position = {a: 1, b: 1, c: 1}
+    mat = Matrix.eye(2)
+    for i in range(1, 401):
```

Comment:
Add here a TODO to change back to walk when we no longer normalize by trajectory size